plugin/Installer/DatabaseChanges.php

```diff
@@ -223,6 +227,24 @@ private function getCreateQueryCache()
 	}
 
 
+	private function deleteMessageFieldApplicantSearchForm()
+	{
+		$prefix = $this->getPrefix();
+		$tableName = $prefix . "oo_plugin_forms";
+		$tableFieldConfig = $prefix . "oo_plugin_form_fieldconfig";
+
+		$rows = $this->_pWPDB->get_results("SELECT `form_id` FROM {$tableName} WHERE form_type = 'applicantsearch'");
+
+		foreach ($rows as $applicantSearchFormId) {
+			$allFieldMessages = $this->_pWPDB->get_results("SELECT form_fieldconfig_id FROM $tableFieldConfig
+										WHERE `fieldname` = 'message'
+										AND `form_id` = '{$applicantSearchFormId->form_id}'");
```

Comment:
Please add proper escaping here (`$applicantSearchFormId->form_id`) and please also rename `$applicantSearchFormId`